models/User.js

```diff
@@ -5,9 +5,9 @@ const userSchema = new Schema({
 	isOwner: Boolean,
 	isPlayer: Boolean,
 	isAdmin: Boolean,
-	ign: String,
 	elo: Number,
-	team: String
+	team: String,
+  accounts: [Schema.Types.ObjectId]
```

Comment:
Here too, either use the player names as a [String] or add ObjectIds to the players.